lib/matplotlib/font_manager.py

```diff
@@ -929,6 +940,45 @@ def set_fontconfig_pattern(self, pattern):
             else:
                 getattr(self, "set_" + key)(val)
 
+    def get_math_fontfamily(self):
+        """
+        Return the name of the font family used for math text.
+        The default font is :rc:`mathtext.fontset`.
+        """
+
+        if self._math_fontfamily is None:
+            return rcParams['mathtext.fontset']
+        return self._math_fontfamily
+
+    def set_math_fontfamily(self, fontset):
```

Comment:
In agreement with the changed naming, the parameter should be named `family` or `fontfamily`, not `fontset`.